web/styles/app_components.css

```diff
@@ -762,11 +772,19 @@ div.overlay {
     border-left: 0;
     border-radius: 4px;
     border-spacing: 0;
+    width: 100%;
+    margin-bottom: 20px;
 
     & th,
     td {
         border-left: 1px solid hsl(0deg 0% 87%);
+        border-top: 1px solid hsl(0deg 0% 87%);
         padding: 4px 5px;
+        text-align: left;
+    }
+
+    thead tr:first-child th {
+        border-top: 0;
```

Comment:
We can probably deduplicate the common CSS for `.table-striped` and `.table-bordered` selectors.